universum.cabal

```diff
@@ -13,11 +13,11 @@ category:            Prelude
 stability:           stable
 build-type:          Simple
 bug-reports:         https://github.com/serokell/universum/issues
-tested-with:         GHC == 8.6.5
-                   , GHC == 8.8.4
+tested-with:         GHC == 8.8.4
                    , GHC == 8.10.7
                    , GHC == 9.0.1
                    , GHC == 9.2.2
```

Comment:
Should we perhaps bump patch-versions of older GHC versions to 9.0.2 and 9.2.8 respectively while we're here?

Reply:
I personally don't have an opinion on this, let me try if it is easy.